ansible_runner/runner.py

```diff
@@ -140,6 +144,18 @@ def run(self):
             for k, v in self.config.env.items()
         }
 
+        # Prepare to collect performance data
+        if self.config.resource_profiling:
+            cgroup_path = '{0}/{1}'.format(self.config.resource_profiling_base_cgroup, self.config.ident)
+            cmd = 'cgcreate -g cpuacct,memory,pids:{}'.format(cgroup_path)
+            proc = Popen(cmd, stdout=PIPE, stderr=PIPE, shell=True)
+            _, stderr = proc.communicate()
+            if proc.returncode:
+                # Unable to create cgroup, disable profiling
+                logger.warning('Unable to create cgroup: {}'.format(stderr))
```

Comment:
Wondering if `logger.error` might make more sense.

Reply:
Yep, raise the severity of this.